### PR TITLE
add support for model endpoint

### LINKED ISSUE #26: Add support  for model endpoint
Even though the readme states that one can configure an endpoint:

image

there is no code to actually support this and the endpoint is therefore fixed: 

https://github.com/IBM/MAX-Question-Answering-Web-App/blob/master/app.py#L34

### PR DESCRIPTION
**Pull request by bdwyer2**
Resolves #26

### CODE REVIEW

**Review by ptitzler — DISMISSED** [review]
LGTM, if you've confirmed that this works as expected and resolves our Kubernetes deployment issue.

**Comment by bdwyer2** [comment]
> LGTM, if you've confirmed that this works as expected and resolves our Kubernetes deployment issue.

Even if this doesn't fix the k8 issue, doesn't this PR still fix a bug?

File: app.py [hunk 1]
```diff
@@ -17,6 +17,12 @@
 from flask import Flask, render_template, request, jsonify
 from chatbot import get_opening_message, get_choice, get_topic, match, narrow, ask, end
 import json
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--port", default=8000)
```
Comment (ptitzler) [inline app.py:23]: The ports needs to be `8088` or to match the port numbers in max-question-answering-web-app.yaml, no?

**Review by ptitzler — CHANGES_REQUESTED** [review]
The port numbers in `Dockerfile`, *.py`, and the kubernetes yaml file need to be consistent. They are currently 8000 and 8088.

**Review by ptitzler — DISMISSED** [review]
lgtm

**Review by ptitzler — APPROVED** [review]
lgtm

**Pull request merged**